src/coffee/directives/api/models/child/marker-child-model.coffee

```diff
@@ -145,10 +145,10 @@ angular.module('uiGmapgoogle-maps.directives.api.models.child')
         return if @isNotValid(scope) or !@gObject?
         @renderGMarker doDraw, =>
           oldValue = @gObject.getIcon()
```

Comment:
Is there a fix on master as well? Nice catch. This was probably from bad merging. Which is why I want 2.1 on master sooner than later.